src/scss/forms/_datagrid.scss

```diff
@@ -0,0 +1,5 @@
+.formio-component-datagrid {
+  td {
+    vertical-align: baseline;
```

Comment:
Thanks @shawnbot ! I went with the second suggestion since seems like it is a little bit more future proof. Can you review again when you have a chance. Also I updated to base to merge into the 9.2.2 release, so hopefully this can go out with that release? Let me know what you think. Thanks!